workflow-testing/src/main/java/com/squareup/workflow1/testing/RenderTester.kt

```diff
@@ -203,6 +206,8 @@ RealRenderTester(this, props, initialState)
  */
 public abstract class RenderTester<PropsT, StateT, OutputT, RenderingT> {
 
+  protected var explicitWorkerExpectationsRequired: Boolean = false
```

Comment:
There's no reason for this to exist in the base class, the only reason this isn't an interface is because we needed to add some inline methods or something. Please keep as much implementation and logic out of this class as possible.

Reply:
I was wondering about that. I thought since this *was* an abstract class I might as well share the implementation of this since it was simple and should be consistent.